sdk/eventhub/Azure.Messaging.EventHubs/tests/Producer/TransportProducerPoolTests.cs

```diff
@@ -0,0 +1,323 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Messaging.EventHubs.Authorization;
+using Azure.Messaging.EventHubs.Core;
+using Azure.Messaging.EventHubs.Producer;
+using Moq;
+using NUnit.Framework;
+
+namespace Azure.Messaging.EventHubs.Tests
+{
+    /// <summary>
+    ///   The suite of tests for the <see cref="TransportProducerPool" /> class.
+    /// </summary>
+    ///
+    [TestFixture]
+    public class TransportProducerPoolTests
+    {
+        /// <summary>
+        ///   The pool periodically removes and closes expired producers.
+        /// </summary>
+        ///
+        [Test]
+        public void ItRemovesExpiredProducers()
```

Comment:
Nothing wrong with the naming that you've used, but it reads differently than the other tests in the library.   I don't feel very strongly about it, but it may be nice to align the pattern a bit closer to the existing tests.

@kinelski :   Thoughts?

Reply:
As far as I am concerned, good point

Reply:
I changed the names of the existing tests in:

- ```TransportProducerPoolRemovesaExpiredItems```
- ```TransportProducerPoolRefreshesAccessedItems```
- ```PoolItemsAreRefreshedOnDisposal```
- ```TransportProducerPoolTracksAProducerUsage```